.travis.yml

```diff
@@ -1,5 +1,6 @@
 # Python project, check against all supported things.
 language: python
+os: osx
```

Comment:
Is this really all it takes to enable MacOS on Travis?

Reply:
Oh, I commited that after pull request. This just was a test